spec/ics-24-host-requirements/README.md

```diff
@@ -0,0 +1,103 @@
+---
+ics: 24
+title: Host State Machine Requirements
+stage: draft
+category: ibc-core
+requires: 2
+required-by: 2, 3, 4, 5, 18
+author: Christopher Goes <cwgoes@tendermint.com>
+created: 2019-04-16
+modified: 2019-04-29
+---
+
+# Synopsis
```

Comment:
I will link to this comment I made on ICS2 which is relevant for this one too (and prob others) https://github.com/cosmos/ics/pull/20/files#r279727245.

Also in github formatting `#` is used for the main title only usually. It might look weird once merged if you use it for sections

Reply:
We've used `#` for top-level titles so far - do you think we should use `##` instead?

There is no main title (the header has special formatting).

Reply:
Yes, I do. I also think we should change the sectioning a bit, as per my comment in the other PR. 

Having both `# Specification` and `# Technical Specification` is confusing

Reply:
:+1: , https://github.com/cosmos/ics/issues/84